GuildedRoseLLC/Items/RemoteItemRepository.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+public class RemoteItemRepository: ItemRepository {
```

Comment:
What is the reason for making this class `public`? My understanding is that the tests are not using this class.

Reply:
resolved